docs/examples/use_cases/tensorflow/yolov4/src/dali/ops.py

```diff
@@ -0,0 +1,212 @@
+import nvidia.dali as dali
```

Comment:
I think you can add a license header, like:
```
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```
As you are contributing to Apache 2.0 project (or other license you think is appropriate and compatible).